synapse/http/server.py

```diff
@@ -242,10 +242,12 @@ async def _async_render(self, request: Request):
         no appropriate method exists. Can be overriden in sub classes for
         different routing.
         """
+        # Treat HEAD requests as GET requests.
```

Comment:
well, ideally we'd only do this if the class didn't have an `_async_render_HEAD`, but given we don't have any such classes, it doesn't really matter

Reply:
I debated on checking this, but it seemed silly to add processing for something we know is always going to be false! If we need this in the future we can add this check.

Reply:
seems fair!